pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/Filter.java

```diff
@@ -67,17 +68,18 @@ public class Filter {
     public static final char CLOSE_PARENTHESIS_CHAR = ')';
     public static final String IS_NULL_TRIMMED = "IS NULL";
 
-    public static String TEXT = "TEXT";
-    public static String OPTIONS = "OPTIONS";
-    public static String DATE = "DATE";
-    public static String DATE_SHORT = "DATE_SHORT";
-    public static String ADDITIONAL_VALUES = "ADDITIONALVALUE";
-    public static String NUMBER = "NUMBER";
-    public static String BOOLEAN = "BOOLEAN";
-    public static String CHECKBOX = "CHECKBOX";
-    public static String COMPOSITE = "COMPOSITE"; //ES type
-    public static String JSON_ARRAY = "JSONARRAY"; //Sample result
-    public static String AGREEMENT = "AGREEMENT";
+    public static final String TEXT = "TEXT";
+    public static final String OPTIONS = "OPTIONS";
+    public static final String DATE = "DATE";
+    public static final String DATE_SHORT = "DATE_SHORT";
+    public static final String ADDITIONAL_VALUES = "ADDITIONALVALUE";
+    public static final String NUMBER = "NUMBER";
+    public static final String BOOLEAN = "BOOLEAN";
+    public static final String CHECKBOX = "CHECKBOX";
+    public static final String COMPOSITE = "COMPOSITE"; //ES type
+    public static final String JSON_ARRAY = "JSONARRAY"; //Sample result
+    public static final String AGREEMENT = "AGREEMENT";
+    public static final String MATRIX = "MATRIX";
```

Comment:
I think `enum` is the best case to use for those constants, there is also enum class of `DSS` `QuestionType` you may find it useful as well.

Reply:
let's create ours then

Reply:
The QuestionType might be moved to dsm-common (or core module) that is common for us.

Reply:
if there is a class in dss-core that dsm needs as well, we should move it to ddp-common!